### ISSUE TITLE
AV1115: Exclude "and" from enum members with [Flags]

### ISSUE THREAD

**Issue by bart-vmware** [bkoelman/CSharpGuidelinesAnalyzer#163]
It's generally fine to use "and" in `enum` member names to indicate the combination of multiple flags. For example:
```c#
[Flags]
public enum DiscoveryMode
{
    Register = 1,
    Query = 2,
    RegisterAndQuery = Register | Query
}
```
In cases like this, no warning should be reported.
